### PR TITLE
Add `@>`

### PR DESCRIPTION
**Pull request by ocharles**
Fixes https://github.com/HeinrichApfelmus/reactive-banana/issues/227

### CODE REVIEW

File: reactive-banana/src/Reactive/Banana/Combinators.hs [hunk 1]
```diff
@@ -295,7 +295,7 @@ switchB b = liftMoment . M . fmap B . Prim.switchB (unB b) . Prim.mapE (unB) . u
 {-----------------------------------------------------------------------------
     Derived Combinators
 ------------------------------------------------------------------------------}
-infixl 4 <@>, <@
+infixl 4 <@>, <@, @>
```
Comment (ocharles) [inline reactive-banana/src/Reactive/Banana/Combinators.hs:298]: I chose `infixl 4`, as that matches `$>` and `*>`.

**Review by mitchellwrosen — APPROVED** [review]
LGTM

**Comment by HeinrichApfelmus** [comment]
Looks good to me as well. Thanks!

**Review by HeinrichApfelmus — COMMENTED** [review]
Looks good to me as well, thanks!

**Pull request merged**